WordPress/src/main/java/org/wordpress/android/ui/ActivityLauncher.java

```diff
@@ -257,9 +257,13 @@ public static void viewQuickStartForResult(Activity activity) {
     }
 
     public static void viewBlogStats(Context context, SiteModel site) {
-        Intent intent = new Intent(context, org.wordpress.android.ui.stats.refresh.StatsActivity.class);
-        intent.putExtra(WordPress.SITE, site);
-        context.startActivity(intent);
+        if (site == null) {
```

Comment:
Can we record this in tracks or somewhere? We should still try to get to the bottom of this, even if its not crashing.

Reply:
sure, I'll add that